### PR TITLE
[OD-303] Configure ckanext-twitter

### PR DESCRIPTION
**Pull request by psbothra**
[OD-303](https://opengovinc.atlassian.net/browse/OD-303) Configure ckanext-twitter

Need to update the extension so that the twitter box does not popup again if the user clicks No to tweet. Currently the popup occurs every single time the user visits the dataset even if they click No.
Test Procedures

-- Install the PR
-- Open the CKAN site and log in as a sysadmin
-- Navigate to the /dataset page and select any dataset
-- Click on Manage
-- Update anything from the selected dataset
-- Once update you will get the twiiter pop-up
-- If user click on No then pop up won't appear until next update

### CODE REVIEW

**Comment by psbothra** [comment]
@injune1123 
1) OD-303 Configure ckanext-twitter 
2) Need to update the extension so that the twitter box does not popup again if the user clicks No to tweet. Currently the popup occurs every single time the user visits the dataset even if they click No.
3) Test Procedures
-- Install the PR 
-- Open the CKAN site and log in as a sysadmin
-- Navigate to the /dataset page and select any dataset
-- Click on Manage
-- Update anything from the selected dataset
-- Once update you will get the twiiter pop-up 
-- If user click on No then pop up won't appear until next update

**Comment by injune1123** [comment]
Good improvement! But the comment should be moved to the top and the title should be changed too. Please look at the images for detailed instructions.  

comment_improvement_2
comment_improvement_1

**Comment by psbothra** [comment]
@injune1123 and @jguo144 : Added the Tweet checkbox on datasets, kindly review it

File: ckanext/twitter/theme/fanstatic/scripts/modules/confirm-tweet.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ ckan.module('confirm-tweet', function ($, _) {
                     _onReceiveSnippet: function (html) {
                         var url    = '/dataset/' + self.options.pkgid + '/tweet';
                         self.modal = $(html);
-                        var form   = self.modal.find('#edit-tweet-form');
+                        var form   = self.modal.find('#edit-tweet-form');
```
Comment (jguo144) [inline ckanext/twitter/theme/fanstatic/scripts/modules/confirm-tweet.js]: I would remove the extra spaces here.

**Pull request merged**